src/js/helpers/Util.js

```diff
@@ -171,7 +171,7 @@ var Util = {
       a != null &&
       b != null &&
       a.length === b.length &&
-      a.every((v, i) => v === b[i]);
+      a.every((v, i) => Util.isEgal(v, b[i]));
```

Comment:
Yep, I left [a note](https://github.com/mesosphere/marathon-ui/pull/629#commitcomment-16012401) on the commit to explain the naming logic!